ssas/service/admin/api_test.go

```diff
@@ -753,6 +753,90 @@ func (s *APITestSuite) TestRegisterDuplicateSystemIP() {
 	_ = ssas.CleanDatabase(group)
 }
 
+func (s *APITestSuite) TestUpdateSystem() {
```

Comment:
This is fine (yay comments!). Better testing practice would be to minimize the number of assertions in a single test--rule of thumb: cover one scenario per test. With something this parallel, though, we're approaching the point where [table-driven tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) become valuable.

Reply:
Agreed, I debated consolidating them like this for a bit, but eventually caved into writing one single test since they were very similar. I can split them up if needed.